docs/development/extensions-contrib/k8s-jobs.md

```diff
@@ -66,6 +66,7 @@ Additional Configuration
 |`druid.indexer.runner.javaOptsArray`|`JsonArray`|java opts for the task.|`-Xmx1g`|No|
 |`druid.indexer.runner.labels`|`JsonObject`|Additional labels you want to add to peon pod|`{}`|No|
 |`druid.indexer.runner.annotations`|`JsonObject`|Additional annotations you want to add to peon pod|`{}`|No|
+|`druid.indexer.runner.peonMonitors`|`JsonArray`|An override for the `druid.monitoring.monitors`, For the situation you have monitors setup, and do not want to inherit those from the overlord.|`[]`|No|
```

Comment:
Nit: It would be useful to mention this property in the [extension specific docs](https://druid.apache.org/docs/latest/development/extensions-contrib/k8s-jobs.html) as well.